src/inspector/models/popups/harppedal/harppedalpopupmodel.h

```diff
@@ -23,13 +23,13 @@
 #ifndef MU_NOTATION_HARPPEDALPOPUPMODEL_H
 #define MU_NOTATION_HARPPEDALPOPUPMODEL_H
```

Comment:
Please change this to `MU_INSPECTOR_HARPPEDALPOPUPMODEL_H` (also at the bottom of the file; and similarly in the other popup models)